src/ntfs/read.rs

```diff
@@ -245,7 +248,7 @@ where
     collection
         .index_mut(idx)
         .comment_links_mut()
-        .push(comment_link.comment_id);
+        .push(comments.get_idx(&comment_link.comment_id).unwrap());
```

Comment:
This `unwrap` might fail if the data is bad, thus you should handle error. @prhod log or fail?

Reply:
As discussed together, we can fail for the moment. We will be permissive only if/when needed.

Reply:
@benoit-bst You must then return a `Result` in this function